microsoft-edge/devtools-guide-chromium/whats-new/2023/03/devtools-111.md

```diff
@@ -0,0 +1,155 @@
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.date: 03/15/2023
+---
+# What's New in DevTools (Microsoft Edge 111)
+
+[!INCLUDE [Microsoft Edge team note for top of What's New](../../includes/edge-whats-new-note.md)]
+
+
+<!-- ====================================================================== -->
+## Remote debugging of Microsoft Edge, PWAs, and WebView2 controls on Xbox and HoloLens devices
+
+<!-- Subtitle: Get the Remote Tools for Microsoft Edge app from the Microsoft Store to set up remote debugging today! -->
+
+The **Remote Tools for Microsoft Edge** app is now available on Xbox and HoloLens devices.  The **Remote Tools for Microsoft Edge** app is a plug-in for the **Windows Device Portal for Desktop** that enables remote debugging for tabs in Microsoft Edge, Progressive Web Apps (PWAs), and WebView2 controls.  By using remote debugging, you can run web content on a remote device and debug it right from your development machine.
+
+![The Remote Tools for Microsoft Edge app in the Microsoft Store](./devtools-111-images/remote-tools-in-the-microsoft-store.png)
+
+With remote debugging enabled, you don't need to open DevTools on either HoloLens or Xbox devices any more.  Instead, you can connect to the remote Xbox or HoloLens device using the Device portal, open the `edge://inspect` page, and then select the `target:` (tab, PWA, or WebView2 control) that you want to use with DevTools.
+
+To give us feedback on the remote debugging flow for Microsoft Edge, leave a comment in [Issue 136: Remote debugging Xbox and HoloLens devices is now available with the latest version of the Remote Tools for Microsoft Edge app](https://github.com/MicrosoftEdge/DevTools/issues/136).
+
+See also:
+* [Remotely debug Windows devices](../../../remote-debugging/windows.md)
+* [Windows Device Portal for Desktop](/windows/uwp/debug-test-perf/device-portal-desktop)
+* [Remote debugging WebView2 WinUI 2 (UWP) apps with Remote Tools for Microsoft Edge](../../../../webview2/how-to/remote-debugging.md)
+
+
+<!-- ====================================================================== -->
+## Device Mode toolbar has emulation controls for themes and vision deficiencies
+
+<!-- Subtitle: Use the new Eyedropper and Eye icons from the Device Mode toolbar to test how your web content responds to light, dark, and high contrast themes and vision deficiencies like color blindness. -->
+
+In previous versions of Microsoft Edge, to emulate different themes or vision deficiencies, you would use the **Rendering** tool — for example, to emulate Achromatopsia, which is when the user is unable to perceive any color, which reduces all color to a shade of grey.
+
+In Microsoft Edge 111, you can now access these themes and vision deficiencies emulation controls directly from the **Device Mode** toolbar, in addition to using the **Rendering** tool.
```

Comment:
Feels like we have too many plurals going on here. How about:
```suggestion
In Microsoft Edge 111, you can now access these theme and vision deficiency emulation controls directly from the **Device Mode** toolbar, in addition to using the **Rendering** tool.
```

since controls can be the plural word here. Multiple of these controls exist for emulating theme and vision deficiency